src/xrpld/app/tx/detail/PayChan.cpp

```diff
@@ -593,24 +536,8 @@ PayChanClaim::doApply()
             (txAccount == src && (sled->getFlags() & lsfDisallowXRP)))
             return tecNO_TARGET;
 
-        // Check whether the destination account requires deposit authorization.
-        if (ctx_.tx.isFieldPresent(sfCredentialIDs))
-        {
-            if (Credentials::removeExpired(view(), ctx_.tx, j_))
-                return tecEXPIRED;
-        }
-        else if (depositAuth && (sled->getFlags() & lsfDepositAuth))
```

Comment:
Why is this check removed?

Reply:
`depositAuth` - same as above
`sled->getFlags() & lsfDepositAuth` - this check is present in `preauth::preclaim`

Reply:
fixed